File: hard/license/license.py
def time(mi_name,n_agents,people):

	names_list = people.split()
	names_list.append(mi_name)
	names_list.sort(key= str.lower) #Ordenamos la lista de manera alfabeticamente

	count = 20
	count2 = 0

	for i in range(len(names_list)):
		if names_list[i] == mi_name:
			break

		count2 += 1
		if count2 == n_agents:
			count += 20
			count2 = 0
	print(count)
		
mi_name = input()
n_agents = int(input())
people = input()

time(mi_name,n_agents,people)

File: hard/license/license.php
<!DOCTYPE html>
<html>
<head>
	<title>Password</title>
</head>
<body>
	<form  method="post">
		<label for="mi_name">Mi name : </label>
		<input type="text"  name="mi_name">
		<br>
		<label for="agents">Number of agents</label>
		<input type="text" name="agents">
		<br>
		<label for="people">Names list</label>
		<input type="text" name="people">
		<br>
		<input type="submit" value="Action" >

	</form>
	<br>
	<?php

		function minutes($mi_name,$agents,$people){

			if ($mi_name == NULL || $agents == NULL || $people == NULL ){
				return("");
			}

			$names_list = explode(" ",$people); 
			array_push ( $names_list , $mi_name );
			sort($names_list, SORT_NATURAL | SORT_FLAG_CASE);

			$count = 20;
			$count2 = 0;

			$posicion = array_search($mi_name,$names_list);

    		$var1= ($posicion+1)/$agents;//2/5 es 2 a menos que use esto
    		$count = ($var1*20)+($var1*20)%20 ;
			echo $count;

		}

	    $mi_name = $_POST["mi_name"];
	    $agents = $_POST["agents"];
	    $people = $_POST["people"];

	    minutes($mi_name,$agents,$people);

	?>

</body>
</html>
		
